### PR TITLE
PTCD-722 Remove Old Help Page

### PR DESCRIPTION
**Pull request by jameslawesynergy**
Removed old /home/help page, updated links to point at /contact and redirected legacy route to contact
Added [AllowDeactivatedProvider] to /contact route

### CODE REVIEW

File: src/Dfc.CourseDirectory.WebV2/Features/Home/HomeController.cs [hunk 1]
```diff
@@ -28,10 +28,9 @@ public IActionResult Index([FromServices] ICurrentUserProvider currentUserProvid
             }
         }
 
-        [HttpGet("help")]
         [HttpGet("Home/Help", Order = 99)]
         [AllowDeactivatedProvider]
-        public IActionResult Help() => View();
+        public IActionResult Help() => RedirectToAction("Index", "Contact");
```
Comment (jameslawesynergy) [inline src/Dfc.CourseDirectory.WebV2/Features/Home/HomeController.cs:33]: Pondering if we should do this sort of thing in middleware?

Comment (gunndabad) [reply src/Dfc.CourseDirectory.WebV2/Features/Home/HomeController.cs:33]: I think this is fine

**Review by gunndabad — APPROVED** [review]

**Review by soharadesign — APPROVED** [review]

File: src/Dfc.CourseDirectory.WebV2/Features/Home/HomeController.cs (hunk 1 above)
Comment (gunndabad) [reply src/Dfc.CourseDirectory.WebV2/Features/Home/HomeController.cs:33]: `Order` parameter is no longer required, though

Comment (jameslawesynergy) [reply src/Dfc.CourseDirectory.WebV2/Features/Home/HomeController.cs:33]: Good spot. Fixed.

Comment (gunndabad) [reply src/Dfc.CourseDirectory.WebV2/Features/Home/HomeController.cs:33]: Unrelated to this but we should maybe consider adding `[AllowDeactivatedProvider]` to `ContractController.Index` too

Comment (jameslawesynergy) [reply src/Dfc.CourseDirectory.WebV2/Features/Home/HomeController.cs:33]: Closely related enough. Fixed.

File: src/Dfc.CourseDirectory.WebV2/Features/Home/HomeController.cs [hunk 2]
```diff
@@ -28,10 +28,16 @@ public IActionResult Index([FromServices] ICurrentUserProvider currentUserProvid
...
 
-        [HttpGet("help")]
-        [HttpGet("Home/Help", Order = 99)]
+        [HttpGet("contact")]
         [AllowDeactivatedProvider]
-        public IActionResult Help() => View();
+        public IActionResult Contact()
+        {
+            return View();
+        }
```
Comment (gunndabad) [inline src/Dfc.CourseDirectory.WebV2/Features/Home/HomeController.cs:36]:
```suggestion
        public IActionResult Contact() => View();
```

**Pull request merged**